packages/react/src/patterns/sub-patterns/ContentSection/ContentSection.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import classNames from 'classnames';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * ContentArrayGroup Component, for use with content arrays
+ *
+ * @param {object} props props object
+ * @param {string} props.heading  Heading objects
+ * @param {string} props.theme theme name
+ * @param {*} props.children JSX Components
+ * @param {string} props.className optional class to be applied to the containing node
+ * @returns {*} JSX ContentSection component
+ */
+const ContentSection = ({ heading, theme, children, customClassName }) => {
+  const className = classNames(`${prefix}--content-section`, customClassName);
```

Comment:
I don't think this line is needed. Either that, or put it below `_setTheme`, and add that piece here instead. That way you only use `classNames` once.